cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -1185,4 +1188,35 @@ public ServiceAccount generateServiceAccount() {
                 .endMetadata()
             .build();
     }
+
+
+    /**
+     * Adds the supplied list of container environment variables {@see io.strimzi.api.kafka.model.ContainerEnvVar} to the
+     * supplied list of fabric8 environment variables {@see io.fabric8.kubernetes.api.model.EnvVar}, checking first if the
+     * environment variable key has already been set in the existing list and then converts them. If a key is already in
+     * use then the container environment variable will not be added to the environment variable list and a warning will
+     * be logged.
+     *
+     * @param existingEnvs The list of fabric8 environment variable object that will be modified.
+     * @param containerEnvs The list of container environment variable objects to be converted and added to the existing
+     *                      environment variable list
+     **/
+    protected void addContainerEnvsToExistingEnvs(List<EnvVar> existingEnvs, List<ContainerEnvVar> containerEnvs) {
```

Comment:
I wonder if this method would better fit the `ModelUtils` class.

Reply:
It uses the `AbstractModel.buildEndVar` method which we could call statically from within `ModelUtils` but that adds coupling between those two classes then?

Reply:
Ok, I guess in that case it makes sense to leave it in `AbstractModel`. Thanks.